WordPress/Classes/Models/WPAccount.m

```diff
@@ -37,13 +35,9 @@ - (void)prepareForDeletion
 
     [_xmlrpcApi.operationQueue cancelAllOperations];
     
-    self.password = nil;
-    
     // Fix: Let's make sure we only nuke the authToken when needed. That is, if the deleted account is dotcom.
     // Otherwise, removing a self hosted account with the same username as a dotcom account might end up nuking the token.
```

Comment:
I assume this comment is no longer valid since we are removing the related code.